source/edge-server/configure.txt

```diff
@@ -441,25 +353,43 @@ Upgrade the Edge Server Version
 As the Edge Server evolves, you may want to upgrade your Edge Server 
 version to take advantage of new functionality.
 
-Prerequisites
-~~~~~~~~~~~~~
-
-Upgrading the Edge Server version requires the host machine to have 
-`jq <https://jqlang.github.io/jq/>`__ installed. If you do not already
-have ``jq`` installed, running an Edge Server upgrade command tells you 
-to install ``jq``.
-
 .. _edge-server-upgrade-commands:
 
 Upgrade Edge Server Commands
```

Comment:
These subheadings don't really fit this page. Maybe format this similar to the "Install and Configure the Edge Server Instance" section?

> Upgrade the Edge Server Version
> -------------------------------
> 
> .. procedure 
> 
>    .. step:: Run the Upgrade Command
>    
>    < move the Upgrade Edge Server Commands section content here >
>    
>    .. step:: Deploy Changes (or whatever...) 
>    
>    < steps 2-5 continue here >